### PR TITLE
Adding  update_gofmt.sh

### PR DESCRIPTION
**Pull request by Rajat-0**
[WIP]Adding update-gofmt script as part of go format tools

### CODE REVIEW

File: vertical-pod-autoscaler/hack/update-gofmt.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (Rajat-0) [reply vertical-pod-autoscaler/hack/update-gofmt.go]: Thanks for the review, I have updated the PR as per your comment

**Comment by losipiuk** [comment]
Thanks!

/approve
/lgtm

**Pull request merged**